configs/peer-products.config.php

```diff
@@ -0,0 +1,6 @@
+<?php
+$CONFIG = [
+  'ionos_peer_products' => [
+	'ionos_webmail_target_link' => 'https://email.ionos.fr/',
```

Comment:
I feel like, it would make more sense to map the links to the market here. Something like this:
```
'ionos_webmail_target_link' => [
    'FR' => 'https://email.ionos.fr/',
]
```
So that we can get the link from the config according to market. Right now we only have `FR`, but in the future we'll probably have more. What do you think?